cmd/version.go

```diff
@@ -39,13 +44,53 @@ func normalizeVersion(v string) string {
 	return v
 }
 
+// getServerVersion attempts to fetch the server version from the configured server.
+// Returns empty string if unable to fetch.
```

Comment:
A better practice is for the function to return `(string, bool)`.
If the func successfully fetched server version, then it returns `(metadata.Version, true)`
Otherwise, it returns `("", false)`.

The code should then check for the bool value and if true, print the server version. If false, print the error instead.